src/strange_poker.c

```diff
@@ -41,10 +43,29 @@ int main(int argc, char *argv[])
                 ui->messageUser( ui->data, &ctx, "No players to play!" );
                 continue;
             }
+            
+            time_t t;
+            srand((unsigned) time(&t));
```

Comment:
srand works with time_t* param as NULL. should be `srand((unsingned int)time(NULL));`.
Also re-rolling seed after every game is unneded and should be done once at the start of the program.